lib/index.js

```diff
@@ -272,10 +272,23 @@ internals.getParamsData = function (param, name, typeName) {
         };
     }
 
-    var type;
+    var originalType = param.type,
+        type;
     if (param.valids && param.valids.some(Joi.isRef)) {
         type = 'reference';
     }
+    else if (param.valids && param.valids.length === 1) {
+        // Simplify the single valid case into a constant display, i.e. {param: "value"}
+        type = internals.getExistsValues(type, param.valids);
+    }
+    else if (!name && param.valids) {
```

Comment:
Generally complex templates are a bad thing. The data generator (i.e. this) should have the complexity and the template have as little logic as possible, at least when using handlebars. These conditionals need to be expressed somewhere and it's most elegant in javascript vs. the templates that may require duplicating the context, etc.